samples/list_vlan_in_portgroups.py

```diff
@@ -0,0 +1,160 @@
+Script to list all portgroup vlan on existing or selected dvswitch
+This is for demonstration and reference purposes.
+Testing ok in Single datacenter, multiple dvswitch instances
+Based on getvnicinfo.py and py-vminfo.py
+
+To improve:
+Error handling
+"""
+
+
+from __future__ import print_function
+from pyVim.connect import SmartConnect, Disconnect
+from pyVmomi import vim
+import atexit
+import argparse
+import getpass
+import ssl
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description='Arguments for talking to vCenter')
+
+    parser.add_argument('-s', '--host',
+                        required=True,
+                        action='store',
+                        help='vSpehre service to connect to')
+
+    parser.add_argument('-o', '--port',
+                        type=int,
+                        default=443,
+                        action='store',
+                        help='Port to connect on')
+
+    parser.add_argument('-u', '--user',
+                        required=True,
+                        action='store',
+                        help='User name to use')
+
+    parser.add_argument('-p', '--password',
+                        required=False,
+                        action='store',
+                        help='Password to use')
+
+    parser.add_argument('-d', '--datacenter',
+                        required=True,
+                        help='name of the datacenter')
+
+    parser.add_argument('-dvs', '--dvswitch',
+                        required=False,
+                        help='name of the dvswitch',
+                        default='all')
+
+    parser.add_argument('-c', '--cert_check_skip',
+                        required=False,
+                        action='store_true',
+                        help='skip ssl certificate check')
+
+    args = parser.parse_args()
+    return args
+
+
+def get_obj(content, vimtype, name):
+    obj = None
+    container = content.viewManager.CreateContainerView(content.rootFolder,
+                                                        vimtype, True)
+    for c in container.view:
+        if c.name == name:
+            obj = c
+            break
+    return obj
+
+
+def get_all_objs(content, vimtype, folder=None, recurse=True):
+    if not folder:
+        folder = content.rootFolder
+
+    obj = {}
+    container = content.viewManager.CreateContainerView(folder,
+                                                        vimtype, recurse)
+    for managed_object_ref in container.view:
+        obj.update({managed_object_ref: managed_object_ref.name})
+    return obj
+
+
+def main():
+    args = get_args()
+    if args.password:
+        password = args.password
+    else:
+        password = getpass.getpass(prompt='Enter password for host %s and '
+                                   'user %s: ' % (args.host, args.user))
+    if args.cert_check_skip:
+        context = ssl._create_unverified_context()
+        si = SmartConnect(host=args.host,
```

Comment:
SmartConnectNOSSL() method can be used instead of creating context here.

Reply:
done!

Reply:
Looks like this change is not done. Please can you check?